### ISSUE TITLE
Title not centered

### ISSUE THREAD

**Issue by Korczynska** [unawe/spaceawe#91]
Hi,

I just spotted something on the page : 
http://www.space-awareness.org/fr/about/

The text appearing in the big picture is not centered inside the image like the other pages.
Could it be ? 

Thanks,

Audrey

**Comment by vehrlich** [comment]
Hi @Korczynska do you mean vertically  ?
It's centred only horizontally. It's padded vertically manually. 
Like on this page http://www.space-awareness.org/fr/skills/

**Comment by Korczynska** [comment]
Yes, like this page, vertically. So should I do something in the backoffice to make it the same centered vertically? 
Thank you!

**Comment by vehrlich** [comment]
@Korczynska this should be style thing. I'll centered it.

**Comment by vehrlich** [comment]
@Korczynska giving it up for now. It looks like html/css structure is little bit more complicated because there is a header (which disappears after scrolling)
In fact text is in the center (see attached picture)
If there is any graphic or css guru, tell me please.
![centering](https://cloud.githubusercontent.com/assets/1701829/23301321/a16e2aea-fa89-11e6-8429-fdc78c9b7838.png)

**Comment by Korczynska** [comment]
Do you need a graphic designer? Shall we ask to Paulo?
